### ISSUE TITLE
Volos guide lizard folk ability missing

### ISSUE THREAD

**Issue by mark-drury** [Sklore/HL_DD_5e_Colab#589]
The Volos guide lizard folk ability hungry jaws is missing from the tracked resources list, it adds temp hit points on a bonus action bite attack.

**Comment by Sklore** [comment]
This will be fixed in the next release. Temp HP will need to be added manually. Updated description to state this.
